2023-08-31 22:52:55,311 INFO    MainThread:7344 [wandb_setup.py:_flush():76] Current SDK version is 0.15.9
2023-08-31 22:52:55,311 INFO    MainThread:7344 [wandb_setup.py:_flush():76] Configure stats pid to 7344
2023-08-31 22:52:55,311 INFO    MainThread:7344 [wandb_setup.py:_flush():76] Loading settings from /Users/fatmaoztel/.config/wandb/settings
2023-08-31 22:52:55,311 INFO    MainThread:7344 [wandb_setup.py:_flush():76] Loading settings from /Users/fatmaoztel/Desktop/Fatima Projects/CTR-prediction/CTR-prediction/wandb/settings
2023-08-31 22:52:55,311 INFO    MainThread:7344 [wandb_setup.py:_flush():76] Loading settings from environment variables: {}
2023-08-31 22:52:55,311 INFO    MainThread:7344 [wandb_setup.py:_flush():76] Applying setup settings: {'_disable_service': False}
2023-08-31 22:52:55,311 INFO    MainThread:7344 [wandb_setup.py:_flush():76] Inferring run settings from compute environment: {'program': '<python with no main file>'}
2023-08-31 22:52:55,311 INFO    MainThread:7344 [wandb_init.py:_log_setup():524] Logging user logs to /Users/fatmaoztel/Desktop/Fatima Projects/CTR-prediction/CTR-prediction/wandb/run-20230831_225255-017q6pye/logs/debug.log
2023-08-31 22:52:55,311 INFO    MainThread:7344 [wandb_init.py:_log_setup():525] Logging internal logs to /Users/fatmaoztel/Desktop/Fatima Projects/CTR-prediction/CTR-prediction/wandb/run-20230831_225255-017q6pye/logs/debug-internal.log
2023-08-31 22:52:55,311 INFO    MainThread:7344 [wandb_init.py:_jupyter_setup():470] configuring jupyter hooks <wandb.sdk.wandb_init._WandbInit object at 0x162c6d650>
2023-08-31 22:52:55,311 INFO    MainThread:7344 [wandb_init.py:init():564] calling init triggers
2023-08-31 22:52:55,311 INFO    MainThread:7344 [wandb_init.py:init():571] wandb.init called with sweep_config: {}
config: {}
2023-08-31 22:52:55,311 INFO    MainThread:7344 [wandb_init.py:init():613] starting backend
2023-08-31 22:52:55,311 INFO    MainThread:7344 [wandb_init.py:init():617] setting up manager
2023-08-31 22:52:55,312 INFO    MainThread:7344 [backend.py:_multiprocessing_setup():106] multiprocessing start_methods=spawn,fork,forkserver, using: spawn
2023-08-31 22:52:55,313 INFO    MainThread:7344 [wandb_init.py:init():623] backend started and connected
2023-08-31 22:52:55,315 INFO    MainThread:7344 [wandb_run.py:_label_probe_notebook():1234] probe notebook
2023-08-31 22:52:55,315 INFO    MainThread:7344 [wandb_run.py:_label_probe_notebook():1244] Unable to probe notebook: 'NoneType' object has no attribute 'get'
2023-08-31 22:52:55,315 INFO    MainThread:7344 [wandb_init.py:init():714] updated telemetry
2023-08-31 22:52:55,329 INFO    MainThread:7344 [wandb_init.py:init():747] communicating run to backend with 60.0 second timeout
2023-08-31 22:52:56,022 INFO    MainThread:7344 [wandb_run.py:_on_init():2180] communicating current version
2023-08-31 22:52:56,310 INFO    MainThread:7344 [wandb_run.py:_on_init():2189] got version response 
2023-08-31 22:52:56,310 INFO    MainThread:7344 [wandb_init.py:init():798] starting run threads in backend
2023-08-31 22:52:56,354 INFO    MainThread:7344 [wandb_run.py:_console_start():2159] atexit reg
2023-08-31 22:52:56,355 INFO    MainThread:7344 [wandb_run.py:_redirect():2014] redirect: wrap_raw
2023-08-31 22:52:56,355 INFO    MainThread:7344 [wandb_run.py:_redirect():2079] Wrapping output streams.
2023-08-31 22:52:56,355 INFO    MainThread:7344 [wandb_run.py:_redirect():2104] Redirects installed.
2023-08-31 22:52:56,355 INFO    MainThread:7344 [wandb_init.py:init():839] run started, returning control to user process
2023-08-31 22:52:57,194 INFO    MainThread:7344 [pyagent.py:run():314] Starting sweep agent: entity=None, project=None, count=None
2023-08-31 22:53:02,724 WARNING MsgRouterThr:7344 [router.py:message_loop():77] message_loop has been closed
2023-08-31 22:53:03,860 ERROR   Thread-7 (_run_job):7344 [jupyter.py:notebook_metadata():224] Failed to detect the name of this notebook, you can set it manually with the WANDB_NOTEBOOK_NAME environment variable to enable code saving.
2023-08-31 22:53:08,965 ERROR   Thread-8 (_run_job):7344 [jupyter.py:notebook_metadata():224] Failed to detect the name of this notebook, you can set it manually with the WANDB_NOTEBOOK_NAME environment variable to enable code saving.
2023-08-31 22:53:14,496 ERROR   Thread-9 (_run_job):7344 [jupyter.py:notebook_metadata():224] Failed to detect the name of this notebook, you can set it manually with the WANDB_NOTEBOOK_NAME environment variable to enable code saving.
2023-08-31 22:53:20,025 ERROR   Thread-10 (_run_job):7344 [jupyter.py:notebook_metadata():224] Failed to detect the name of this notebook, you can set it manually with the WANDB_NOTEBOOK_NAME environment variable to enable code saving.
2023-08-31 22:53:25,019 INFO    MainThread:7344 [jupyter.py:save_ipynb():373] not saving jupyter notebook
